docs/reviewing.md

```diff
@@ -19,6 +20,20 @@ Avoid trailing white-space characters, if possible
 
 If possible, try to avoid mixing single quotes (`'`) and double quotes (`"`) in python code (`conanfile.py`, `test_package/conanfile.py`). Consistency is preferred.
 
+## Subfolder Properties 
+
+When extracting sources or performing out-of-source builds, it is preferable to use property attributes for these variables.
```

Comment:
I still think this is a mistake 😄 

It might be more pythonic, but it is more boilerplate and I don't see any advantage. I can't remember a single case where the `_source_subfolder` or `_build_subfolder` were accidentally changed in recipe code.

Reply:
Very good point.

I think how it's written distract form the "use this variable" ant not something else (which is my main goal writing this)

How about changing the wording?

> when extracting source it is prefered to have a property `_source_subfolder = "source_subfolder"`
> when performing out of source build, it is prefered too...

Reply:
It's a boilerplate indeed. It makes sense for big recipes. They were not changed in any recipe, because they are read-only haha

Reply:
agree, I'd prefer less code for this